src/views/LobbyView.vue

```diff
@@ -20,6 +20,17 @@
         </h1>
       </v-col>
     </v-row>
+    <v-row>
+      <v-col offset="5">
+        <v-switch
+          v-model="gameStore.isRanked"
+          :label="gameStore.isRanked ? 'Ranked' : 'Normal'"
+          data-cy="edit-game-ranked-switch"
+          color="primary"
+          @update:model-value="modeChange"
```

Comment:
let's rename the function
```suggestion
          @update:model-value="setIsRanked"
```